internal/distro/fedora32/distro.go

```diff
@@ -190,9 +190,14 @@ func (t *imageType) BuildPackages() []string {
 func (t *imageType) Manifest(c *blueprint.Customizations,
 	options distro.ImageOptions,
 	repos []rpmmd.RepoConfig,
-	packageSpecs,
-	buildPackageSpecs []rpmmd.PackageSpec,
+	packageSpecSets [][]rpmmd.PackageSpec,
 	seed int64) (distro.Manifest, error) {
+	var packageSpecs []rpmmd.PackageSpec
+	var buildPackageSpecs []rpmmd.PackageSpec
+	if len(packageSpecSets) == 2 {
```

Comment:
This seems brittle. Is is so that you can pass `nil` from tests?

Reply:
It did.